navigator/runtime-fragment/src/main/java/com/freeletics/mad/navigator/fragment/HandleNavigation.kt

```diff
@@ -36,7 +38,9 @@ public fun handleNavigation(fragment: Fragment, navigator: NavEventNavigator) {
     dispatcher.addCallback(fragment, navigator.onBackPressedCallback)
 
     lifecycle.coroutineScope.launch {
-        navigator.collectAndHandleNavEvents(lifecycle, executor, activityLaunchers)
+        lifecycle.repeatOnLifecycle(Lifecycle.State.RESUMED) {
+            navigator.collectAndHandleNavEvents(executor, activityLaunchers)
+        }
```

Comment:
`repeatOnLifecycle` is what `flowOnLifecycle` does internally, so for `Fragment` nothing changes.